File: Tools/Log.cs
using System;

namespace DragonzLandBot
{
    public class Log
    {
        public static void Show(string Game, string Account, string Message, ConsoleColor Color)
        {
            Console.ForegroundColor = ConsoleColor.White;
            Console.Write($"[{DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss")}] ");
            Console.ForegroundColor = ConsoleColor.DarkYellow;
            Console.Write($"[{Game}] ");
            Console.ForegroundColor = ConsoleColor.Cyan;
            Console.Write($"[{Account}] ");
            Console.ForegroundColor = Color;
            Console.WriteLine(Message);
            Console.ResetColor();
        }
    }
}

File: Tools/Log.vb
Public Class Log
    Public Shared Sub Show(ByVal Game As String, ByVal Account As String, ByVal Message As String, ByVal Color As ConsoleColor)
        Console.ForegroundColor = ConsoleColor.White
        Console.Write($"[{Date.Now.ToString("yyyy-MM-dd HH:mm:ss")}] ")
        Console.ForegroundColor = ConsoleColor.DarkYellow
        Console.Write($"[{Game}] ")
        Console.ForegroundColor = ConsoleColor.Cyan
        Console.Write($"[{Account}] ")
        Console.ForegroundColor = Color
        Console.WriteLine(Message)
        Console.ResetColor()
    End Sub
End Class
